src/sql/src/statements/insert.rs

```diff
@@ -87,8 +110,14 @@ fn sql_exprs_to_values(exprs: &Vec<Vec<Expr>>) -> Result<Vec<Vec<Value>>> {
                 Expr::Value(v) => v.clone(),
                 Expr::Identifier(ident) => {
                     if ident.quote_style.is_none() {
-                        Value::Placeholder(ident.value.clone())
+                        // Special processing for `default` value
+                        if ident.value.to_lowercase() == "default" {
+                            Value::Placeholder(ident.value.clone())
+                        } else {
+                            return None;
```

Comment:
I still prefer the old way of returning `Result`. If this function is called, we should have the contract that `exprs` are all values. Otherwise it's a bug. We can have an meaningful error msg were the contract is broken. For example, you can `ensure!` here to confirm that ident without quotes has to be equal to "default". `Option` can't achieve that.